test/sslapitest.c

```diff
@@ -1038,17 +1038,16 @@ static int ping_pong_query(SSL *clientssl, SSL *serverssl)
     char srec_wseq_after[SEQ_NUM_SIZE];
     char srec_rseq_before[SEQ_NUM_SIZE];
     char srec_rseq_after[SEQ_NUM_SIZE];
-    SSL_CONNECTION *clientsc, *serversc;
-
-    if (!TEST_ptr(clientsc = SSL_CONNECTION_FROM_SSL_ONLY(clientssl))
-        || !TEST_ptr(serversc = SSL_CONNECTION_FROM_SSL_ONLY(serverssl)))
+ 
+    if (!TEST_ptr(clientssl)
+        || !TEST_ptr(serverssl))
         goto end;
 
     cbuf[0] = count++;
-    memcpy(crec_wseq_before, &clientsc->rlayer.wrl->sequence, SEQ_NUM_SIZE);
-    memcpy(srec_wseq_before, &serversc->rlayer.wrl->sequence, SEQ_NUM_SIZE);
-    memcpy(crec_rseq_before, &clientsc->rlayer.rrl->sequence, SEQ_NUM_SIZE);
-    memcpy(srec_rseq_before, &serversc->rlayer.rrl->sequence, SEQ_NUM_SIZE);
+    memcpy(crec_wseq_before, &clientssl->rlayer.wrl->sequence, SEQ_NUM_SIZE);
+    memcpy(srec_wseq_before, &serverssl->rlayer.wrl->sequence, SEQ_NUM_SIZE);
+    memcpy(crec_rseq_before, &clientsl->rlayer.rrl->sequence, SEQ_NUM_SIZE);
```

Comment:
This typo caused at least one CI failure
```suggestion
    memcpy(crec_rseq_before, &clientssl->rlayer.rrl->sequence, SEQ_NUM_SIZE);
```

Reply:
don't forget this one